src/handlers.rs

```diff
@@ -154,6 +156,13 @@ where
                 tx_builder.unspendable(unspendable);
             }
 
+            if let Some(base64_data) = add_data {
+                let op_return_data = base64::decode(&base64_data).unwrap();
+                tx_builder.add_data(op_return_data.as_slice());
+            } else if let Some(string_data) = add_string {
+                tx_builder.add_data(string_data.as_bytes());
+            }
```

Comment:
This logic will update when we allow both variants..

Reply:
Thank you Raj I'll be working on this also.

Reply:
Because of the `conflict_with` lines above the `--add_string` and `--add_data` parameters won't conflict.